.github/scripts/validate-dist

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+mvn -B clean install -Papache-release,dist,rat -DskipTests -Ddependency-check.skip -Dgpg.skip
```

Comment:
Will it cause the CI to fail in case of CVE's ?

Reply:
it has `-Ddependency-check.skip` ; so that's not being run...
it would be nice to have an opt-in label to run those on any PR (and on master by default) - but that's a longer story